app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala

```diff
@@ -54,6 +54,26 @@ object MoneyUtils {
         else Right(d)
       }
 
+  def validateLimit(key: String, exceedsLimit: BigDecimal => Boolean)(
+    s: String
```

Comment:
Do you need `String` here? From the way you're using it it looks like you can always pass in a `BigDecimal`

Reply:
Key is used to mention the error message prefix. If there is a better way I may take that approach too.

Reply:
I meant the `s` string here - but you might not even need this method if you take a look at my other comment

Reply:
Done.